src/sentry/static/sentry/app/components/events/interfaces/generic.tsx

```diff
@@ -1,14 +1,13 @@
 import PropTypes from 'prop-types';
 import React, {Component} from 'react';
 
-import SentryTypes from 'app/sentryTypes';
 import ButtonBar from 'app/components/buttonBar';
 import Button from 'app/components/button';
 import EventDataSection from 'app/components/events/eventDataSection';
 import KeyValueList from 'app/components/events/interfaces/keyValueList/keyValueList';
 import {t} from 'app/locale';
 
-function getView(view, data) {
+function getView(view: View, data: Record<string, any>) {
```

Comment:
you can reuse the State type here 😉

Reply:
I'll just pick from the state type 👍. I don't really want to mess with functionality for a small typescript improvement.